src/components/general/EntryCommentModal/CommentForm/index.tsx

```diff
@@ -92,13 +90,13 @@ function CommentForm(props: Props) {
 
     const handleFaramValidationSuccess = useCallback((_, newFaramValues) => {
         if (newFaramValues?.text === '') {
-            setFaramValues(oldFaramValues => ({
-                ...oldFaramValues,
+            postComment({
+                ...newFaramValues,
                 text: undefined,
-            }));
+            });
         }
         setPristine(true);
-        postComment();
+        postComment(newFaramValues);
```

Comment:
Why post twice? Was the second postComment intended to be on else block?
Also, setPristine should be called on onSuccess